webapp/src/components/link_preview/preview.css

```diff
@@ -0,0 +1,156 @@
+:root {
+    --light-gray: #6a737d;
+    --blue: #274466;
+    --light-blue: #eff7ff;
+    --github-merged: #6f42c1;
+    --github-closed: #cb2431;
+    --github-open: #28a745;
+    --github-not-planned: #6e7681
+}
+
+@media (min-width: 544px) {
+    .github-preview--large {
+        min-width: 320px;
+    }
+}
+
+/* Github Preview */
+.github-preview {
+    background-color: var(--center-channel-bg-rgb);
+    box-shadow: 0 2px 3px rgba(0,0,0,.08);
+    position: relative;
+    width: 100%;
+    max-width: 700px;
+    border-radius: 4px;
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+}
+
+/* Header */
+.github-preview .header {
+    color: rgba(var(--center-channel-color-rgb), 0.64);
+    font-size: 11px;
+    line-height: 16px;
+    white-space: nowrap;
+}
+
+.github-preview .header a {
+    text-decoration: none;
+    color: rgba(var(--center-channel-color-rgb), 0.64);
+    display: inline-block;
+}
+
+.github-preview .header .repo {
+    color: var(--center-channel-color-rgb);
+}
+
+/* Body */
+.github-preview .body > span {
+    line-height: 1.25;
+}
+
+/* Info */
+.github-preview .preview-info {
+    line-height: 1.25;
+    display: flex;
+    flex-direction: column;
+}
+.github-preview .preview-info > a, .github-preview .preview-info > a:hover {
```

Comment:
We can make this file `scss` them we can use nesting for this selector logic

Reply:
Can I use scss on this project? I thought I couldn't use SCSS because I couldn't find the SCSS file on this repository.